lib/settings.dart

```diff
@@ -39,6 +41,10 @@ class _SettingsState extends State<Settings> {
 
   @override
   Widget build(BuildContext context) {
+    final appInfoChangeNotifier = context.read<AppInfoChangeNotifier>();
```

Comment:
nit: if you only intend to read the app info, there is no need to implement the change notifier, resp. update listeners as this will have no effect here.